site/cypress/e2e/courseMaterials.spec.js

```diff
@@ -782,4 +820,5 @@ describe('Test cases revolving around course material uploading and access contr
             });
         }
     });
+
```

Comment:
this line isn't related to the rest of this PR and should be reverted to prevent potential merge conflicts with other PRs.